### PR TITLE
Final clean up

### PR DESCRIPTION
**Pull request by CharlieMorocz**
<h2>Description</h2>
Removed large chunks of unused code and unnecessary imports, as well as slight modifications to structure and ReadMe.

<h2>Motivation and Context</h2>
Unused code was removed to improve general cleanliness and readability. ReadMe was updated to better reflect the final state of the repo as well as to include a demo of the current final product.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,10 @@
     <img src="src/presentation/assets/NLULogoTransparent.png" alt="Logo" width="200" height="200">
```
Comment (rumncoke-ai) [inline README.md:14]: The Read me file looks great! I love the comments

File: src/presentation/styles/styleSheets/sass/UploadPage.scss [hunk 2]
```diff
@@ -1,153 +1,94 @@
...
+        h3 {
+          font-size: 40vw;
+          top: 50%;
+          transform: translateY(-50%);
+          opacity: .1;
+          color: #fff;
           line-height: 1;
           overflow: hidden;
         }
-        h2{
```
Comment (rumncoke-ai) [inline src/presentation/styles/styleSheets/sass/UploadPage.scss:80]: Good job on removing unnecessary data

File: src/presentation/styles/styleSheets/css/UploadPage.css [hunk 3]
```diff
@@ -129,81 +114,10 @@ html {
...
-.main__form .form-group input::placeholder,
-.main__form .form-group textarea::placeholder {
-  color: transparent;
-}
-
 .main__form .form-group input {
   height: 6rem;
 }
 
-.main__form .form-group.form-message label {
```
Comment (rumncoke-ai) [inline src/presentation/styles/styleSheets/css/UploadPage.css:151]: Great job on dealing with removing the unnecessary data from here

**Review by rumncoke-ai — APPROVED** [review]

**Review by MartinCalcaterra — COMMENTED** [review]
Nice job removing all of those extra imports, I notcied them while I was working on the front end and they were throwing a ton of warnings in the VSCode console.

**Pull request merged**